src/steelshield/gamefabric/introduction.md

```diff
@@ -0,0 +1,60 @@
+# Introduction
+
+## Description
+
+SteelShield is a DDoS mitigation solution. It protects game servers from large-scale DDoS attacks.
+
+SteelShield intercepts network traffic before it reaches your game server. It checks if packets are legitimate by examining:
+- Source address
+- Destination address  
+- Port
+- Packet contents
+
+SteelShield allows only expected traffic through to your game server.
+
+## Protection Levels
+
+::: info
+This describes the situation inside Nitrado's own infrastructure.
+:::
+
+### Level 1 - Reactive Rate Limiting
+
+> No integration needed from game developers.
+
+This protection monitors packet size and packets per second. When it detects unusual network activity, it rate limits traffic to the server.
```

Comment:
When you use “it,” the reader has to decide which of those prior nouns “it” refers to. Even if the intended referent seems obvious to you as the author, the reader might reasonably interpret “it” differently.

Example:

> The controller communicates with the sensor via a serial interface. It stores the data for later use.

Here, “it” could mean:

* the controller, or
* the sensor, or even
* the interface (grammatically possible, though unlikely).

That ambiguity forces the reader to pause and interpret, which is something you want to avoid in technical writing. Also, if the reader only reads the second sentence, they will be even more confused. Ideally each sentence should be explicit and clear with minimal context.